src/css/controls/imports/tooltip.less

```diff
@@ -148,6 +149,7 @@
     padding: 0;
     pointer-events: none;
     user-select: none;
+    will-change: transform;
```

Comment:
`will-change` is not supported in IE: https://developer.mozilla.org/en-US/docs/Web/CSS/will-change

Reply:
Great article and good to know! I didn't know all of that about the `will-change` property.

Reply:
@pajong we have an IE flag, can use translateZ(0) to get a similar effect, but I try not to use it in a non-device-specific way since GPU acceleration eats battery life.